docs/axes/README.md

```diff
@@ -16,7 +16,7 @@ The following properties are common to all axes provided by Chart.js.
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| `display` | `boolean/string` | `true` | Controls the axis global visibility (visible when `true`, hidden when `false`). When `display: 'auto'`, the axis is visible only if at least one associated dataset is visible.
+| `display` | <code>boolean&#124;string</code> | `true` | Controls the axis global visibility (visible when `true`, hidden when `false`). When `display: 'auto'`, the axis is visible only if at least one associated dataset is visible.
```

Comment:
I like the HTML `|` instead of `/` but I think I prefer `boolean/string` (in sources) which make easier to read/edit the markdown code.

Reply:
While I agree that `|` (union) is more correct and TypeScript compatible. So maybe `<code>type1&#124;type2</code>` as you did is just fine.

Reply:
I agree that `/` is easier to edit but `|` looks correct. Markdown just doesn't allow to put it in a table...